core/coreobjects/include/coreobjects/property_impl.h

```diff
@@ -114,6 +115,33 @@ class PropertyImpl : public ImplementationOf<IProperty, ISerializable, IProperty
         checkErrorInfo(validateDuringConstruction());
     }
 
+    PropertyImpl(IPropertyBuilder* propertyBuilder)
```

Comment:
These constructors should now replace the ones using dictionaries imo - those were really only created for use within the `build` method. 

Those dictionary constructors should be removed on all builder-pattern objects.